Blockzilla/AppDelegate.swift

```diff
@@ -19,6 +19,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         #if BUDDYBUILD
             BuddyBuildSDK.setup()
         #endif
+        // Need to get "settings.sendUsageData" this way so that Sentry can be initialized
+        // before getting the Profile.
```

Comment:
This comment needs adjusted for Focus. In Firefox, we wanted Sentry to be set up before we init our `Profile` instance which could fail itself. I don't believe that you have a `Profile` in Focus.